### ISSUE TITLE
Double exports?

### ISSUE THREAD

**Issue by jkhartshorne** [jkhartshorne/KidTalkIssues#63]
I'm still getting two emails with database exports, which contain different information. Maybe one is for staging? But it's hard to tell which is which. I only get one sound export, but I think it's only for staging??

At this point, I think I have my local copy completely confused. Can I please get a *complete* export of the databases so that we're up to date with current status? Thanks!

(And also fix the double export.)

**Comment by alexchippendale** [comment]
@jkhartshorne by complete export you mean all time production db export correct?

Also it sounds like a label distinguishing staging vs production exports is necessary and would also fix #62 ? Can you confirm this? Thanks!

**Comment by jkhartshorne** [comment]
Yes, this is the same as #62. I didn't realize that was still open. A label distinguishing them would definitely work.

But yes, an all time production db export would be super useful. Because at this point, I'm not sure what I have is all and only the production db.

**Comment by alexchippendale** [comment]
Okay that makes sense
